server/api/dbConfig/storyDatabase.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS storyDatabase;
+
+CREATE TABLE storyDatabase (
+    id serial PRIMARY KEY,
+    title VARCHAR,
+    authorName VARCHAR,
+    story VARCHAR
```

Comment:
I think it might be better if there are character limits for the title, the authorName and the story.